templates/task-maven.yaml

```diff
@@ -20,6 +20,12 @@ spec:
     - name: sever_secret
```

Comment:
I see name mismatch its `sever_secret` here and everywhere its used `server_secret`